File: URIS2023-AgilIS-Tim10-main/Status_MicroService/Status_MicroService/Entities/Status.cs
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations;

namespace Status_MicroService.Entities
{
    [Index(nameof(BacklogItemId), IsUnique = true)]
    public class Status
    {
        [Key]
        [Required]
        public Guid IdStatusa  { get; set; }

        public string VrednostStatusa { get; set; }
        public Guid BacklogItemId { get; set; }
    }
}

File: URIS2023-AgilIS-Tim10-main/URIS2023_TempateProject-main/src/app/models/status.ts
export class Status{
    idStatusa: string
    vrednostStatusa: string
    backlogItemId: string
}


export class StatusCreation {
    vrednostStatusa: string
    backlogItemId: string
 
    constructor(vrednostStatusa,backlogItemId ){
     this.vrednostStatusa=vrednostStatusa;
     this.backlogItemId =backlogItemId
    }
 }
 
 export class StatusUpdate {
    idStatusa: string
    vrednostStatusa: string
 
     constructor(idStatusa, vrednostStatusa ){
       this.idStatusa =idStatusa
       this.vrednostStatusa=vrednostStatusa;
        }
    }